i18n/de/docusaurus-plugin-content-docs/current/toolbox/accessibility_indicators/closest_average.md

```diff
@@ -6,24 +6,26 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import thematicIcon from "/img/toolbox/data_management/join/toolbox.webp";
 
-# Heatmap - Closest Average
-A color-coded map to visualize the average travel time to points (such as [POI](../../../further_reading/glossary#point-of-interest-poi "What is a POI?")) from surrounding areas.
+# Heatmap - Durchschnitt Reisezeit
 
-## 1. Explanation
+Eine farbkodierte Karte zur Visualisierung der durchschnittlichen Reisezeit zu Punkten (wie [POI](../../../further_reading/glossary#point-of-interest-poi "Was ist ein POI?")) aus umliegenden Gebieten.
 
-Visualized as a color-coded hexagonal grid, the heatmap takes into account real-world transport and street networks to compute travel times. After specifying a *routing type* (Walk, Bicycle, etc.), *opportunity layer* and *travel time limit*, the result will display a color-coded hexagonal grid for all areas accessible under these conditions. The color scale refers to average travel time.
+## 1. Erklärung
+
+
+Visualisiert als farbkodiertes hexagonales Raster, berücksichtigt die Heatmap reale Transport- und Straßennetze, um Reisezeiten zu berechnen. Nach der Festlegung eines *Routing-Typs* (Gehen, Fahrrad, usw.), einer *Opportunity-Schicht* und eines *Reisezeitlimits* wird das Ergebnis ein farbkodiertes hexagonales Raster für alle unter diesen Bedingungen zugänglichen Bereiche anzeigen. Die Farbschattierung bezieht sich auf die durchschnittliche Reisezeit.
 
 :::info INFO
 
-An `Opportunity layer` contains [geographic point](../../../data/data_types "What are geographic points?") data. Select one or more such layers containing your destination points (opportunities) as input to the heatmap.
+Eine `Opportunity-Schicht` enthält [geografische Punkt](../../../data/data_types "Was sind geografische Punkte?")-Daten. Wählen Sie eine oder mehrere solcher Schichten mit Ihren Zielpunkten (Opportunities) als Eingabe für die Heatmap.
```

Comment:
It looks like even in German, we use the term "Layer" in GOAT. So maybe we should just use Layer here too. Could you do a find-replace for any other locations where "Schicht" was used and switch it back to Layer?

<img width="417" alt="Screenshot 2024-05-27 at 11 25 38 PM" src="https://github.com/goat-community/goat-docs/assets/24190384/138786aa-1e5e-45c8-8e25-2e6affc28649">
<img width="417" alt="Screenshot 2024-05-27 at 11 25 38 PM" src="https://github.com/goat-community/goat-docs/assets/24190384/138786aa-1e5e-45c8-8e25-2e6affc28649">